e2e/react-start/rsc/tests/rsc-deferred.spec.ts

```
@@ -1,5 +1,6 @@
 import { expect } from '@playwright/test'
 import { test } from '@tanstack/router-e2e-utils'
+import { waitForHydration } from './hydration'
```

ESLint

[error] Expected 1 empty line after import statement not followed by another import.

(import/newline-after-import)